nginx_rock/etc/main_location.conf

```diff
@@ -1 +1 @@
-return 404;
+return 404;
```

Comment:
Should a newline be added here?

Reply:
whoops, yes. accidentally overwrote these files and tried to revert, missed new line adn the linter did not pick it up. fixed now